composer.json

```diff
@@ -28,7 +28,7 @@
         "php" : "^7.0",
         "illuminate/config": "~5.1.24|~5.2.0|~5.3.0|~5.4.0",
         "illuminate/database": "~5.1.24|~5.2.6|~5.3.0|~5.4.0",
-        "illuminate/support": "~5.1.24|~5.2.0|~5.3.0|~5.4.0",
+        "illuminate/support": "~5.3.0|~5.4.0",
```

Comment:
`mapWithKeys` is used here: https://github.com/spatie/laravel-activitylog/pull/122/files#diff-6bae372eb15ba8af41415ab509d0982bR61